src/YiiElasticSearch/DataProvider.php

```diff
@@ -34,7 +34,7 @@ class DataProvider extends CDataProvider
     /**
      * @var ResultSet the search result set
      */
-    protected $resultSet;
+    public $resultSet;
```

Comment:
@mikehaertl I've added this an I saw that there is not reason not a reason not be a public. There was no setter since it's populated after a search while the getter wouldn't have any special functionality.